hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -768,9 +768,7 @@ void scheduleRestart(long jobId) {
     void scheduleSnapshot(MasterContext mc, long executionId) {
         long snapshotInterval = mc.jobConfig().getSnapshotIntervalMillis();
         ExecutionService executionService = nodeEngine.getExecutionService();
-        if (logger.isFineEnabled()) {
-            logger.fine(mc.jobIdString() + " snapshot is scheduled in " + snapshotInterval + "ms");
-        }
+        logFine(logger, "%s snapshot is scheduled in %d ms", mc.jobIdString(), snapshotInterval);
```

Comment:
This was done this way because `mc.jobIdString()` isn't a simple getter. Users mostly have this logging off. Again below, multiple times.